Materials2050_Adapter/Materials2050_Adapter.csproj

```diff
@@ -49,6 +49,27 @@
       <Private>false</Private>
       <SpecificVersion>false</SpecificVersion>
     </Reference>
+    <Reference Include="HTTP_Engine">
+      <HintPath>..\..\..\ProgramData\BHoM\Assemblies\HTTP_Engine.dll</HintPath>
```

Comment:
These will need resolving - [BHoMBot](https://bhom.xyz/documentation/DevOps/Bots/BHoMBot/#interacting-with-bhombot) can help you with project-compliance 😄